e2e-env/docker-compose.yml

```diff
@@ -29,6 +29,8 @@ services:
       - ${HOST_PORT_BIND_PRESTASHOP:?See e2e-env/.env.dist}:80
     networks:
       - prestashop
+    extra_hosts:
+      - host.docker.internal:host-gateway
```

Comment:
Why would you need this? For the XDEBUG feature?

Reply:
yes, but it's not right actually, i remove this.